src/Job/JobId.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace SlmQueueBeanstalkd\Job;
+
+use Pheanstalk\Contract\JobIdInterface;
+
+class JobId implements JobIdInterface  {
+    /**
+     * JobId constructor.
+     * @param int $id
+     */
+    public function __construct(int $id = null)
+    {
+        $this->id = $id;
+    }
+
+    /**
```

Comment:
Feel free to remove the PHPDoc on every method call that is allready fully typed :). Or remove useles comments about constructors and such.